Automattic/jetpack
# AI Assistant: Add requirement for one button in forms

Fixes #32220

## Proposed changes:

*

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Add an empty form
* Ask for a form
* Check that it responds with one submit button
